understanding/22/target-size-minimum.html

```diff
@@ -13,6 +13,15 @@ <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section>
 
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make controls easier to activate.</dd>
+            <dt>What to do</dt><dd>Ensure targets meet a minimum size or have sufficient spacing around them.</dd>
+            <dt>Why it's important</dt><dd>Some people with physical impairments cannot click small buttons that are close together.</dd>
```

Comment:
```suggestion
            <dt>Why it's important</dt><dd>Some people with physical impairments find it hard to click small buttons that are close together.</dd>
```

Reply:
I believe the EOWG would like to emphasize inability over difficulty, and there are some people for whom this more of a 'cannot' do rather than a 'not easy'.